code/__DEFINES/_init.dm

```diff
@@ -4,6 +4,11 @@
 	// This exists so that world.Profile() is THE FIRST PROC TO RUN in the init sequence.
 	// This allows us to get the real details of everything lagging at server start.
 	world.Profile(PROFILE_START)
+	#if defined(ENABLE_BYOND_TRACY) && (DM_BUILD == 1589)
```

Comment:
Version locking it because lord knows whats gonna go wrong if I load this on an incompatible version.